request-management-api/request_api/services/eventservice.py

```diff
@@ -61,14 +61,6 @@ def posteventforaxisextension(self, ministryrequestid, extensionids, userid, use
         except BusinessException as exception:            
             self.__logbusinessexception(exception)
 
-    def attachmentevent(self, ministryrequestid, document, userid, event, message):
-        try:
-            attachmenteventresponse = attachmentevent().createattachmentevent(ministryrequestid, message , userid, event)
-            if attachmenteventresponse.success == False:
-                current_app.logger.error("FOI Notification failed for event for attachment= %s" % (document['category']))
-        except BusinessException as exception:            
-            self.__logbusinessexception(exception)
-            
     def postreminderevent(self):
```

Comment:
You should keep this code. refer other events like extension/comment. All these are called in eventservice.